icechunk-python/python/icechunk/store.py

```diff
@@ -332,3 +332,6 @@ def list_dir(self, prefix: str) -> AsyncIterator[str]:
         # listing methods should not be async, so we need to
         # wrap the async method in a sync method.
         return self._store.list_dir(prefix)
+
+    async def getsize(self, key: str) -> int:
```

Comment:
there is a `getsize_prefix` too: https://zarr.readthedocs.io/en/stable/api/zarr/abc/store/index.html#zarr.abc.store.Store.getsize_prefix

Reply:
yeah, see comment in the PR ... I want to try if we can skip implementing those. If performance is not there we can implement.

Reply:
AH ok. I'll push a benchmark, then approve.